spec/thank_you_mailer_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'thank_you_mailer'
+
+RSpec.describe ThankYouMailer do
+  let(:email) { ThankYouMailer.send_email('user@example.com', 'Firstname') }
+
+  it 'requires a from field' do
+    expect(email.from).not_to be(nil)
+  end
+
+  it 'sends an email to the receiver' do
+    expect(email.to).to eq(['user@example.com'])
+  end
+
+  it 'sends a simple text email including the person name' do
+    expect(email.body).to include('Firstname')
```

Comment:
Keep in mind that I think accessing the email in this way is giving you the non-decoded version.. Which in the future means you might have problems with trying to match content and line-wrapping.. Here's an example from the body of real email:

```
 further insight and details.  We are actively looking to build our securit=
y community and I thought that might be of interest to you.  Look forward t=
o speaking to you, I have forwarded an invite to you separate to this email=
.
```

Reply:
Ah, I see what you meant now.

In the future when we have a proper text, this test wont be needed as all this will be done through mandrill or some other service. This is just for the moment. :smiley: